docker-for-windows/faqs.md

```diff
@@ -8,7 +8,7 @@ title: Frequently asked questions (FAQ)
 
 ### How do I get the Stable or the Edge version of Docker Desktop?
 
-A. You can download the Stable version of Docker Desktop from [Docker Hub](https://hub.docker.com/?overlay=onboarding). To download the Edge version, see the [Edge release notes](/docker-for-windows/edge-release-notes/).
+A. You can download the Stable version of Docker Desktop from [Docker Hub](https://hub.docker.com/editions/community/docker-ce-desktop-windows). To download the Edge version, see the [Edge release notes](/docker-for-windows/edge-release-notes/).
```

Comment:
Should we point to the Stable version - https://download.docker.com/win/stable/Docker%20Desktop%20Installer.exe?